### PR TITLE
Add shopify-polaris-docs extension

### PR DESCRIPTION
**Pull request by Rafi993**
## Description

* Raycast extension to search Shopify Polaris (Shopify's UI library https://polaris.shopify.com/) doc directly from Raycast
* The doc folder inside assets contain the doc in markdown format so that it can be searched and displayed even when offline (resulting in lot of file changes shown eventhough actual extension code is limited to 3 files)

## Screencast

demo1
demo2

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/shopify-polaris-docs/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Shopify-Polaris-Docs Changelog
```
Comment (pernielsentikaer) [inline extensions/shopify-polaris-docs/CHANGELOG.md:1]:
```suggestion
# Shopify Polaris Docs Changelog
```

File: extensions/shopify-polaris-docs/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Shopify-Polaris-Docs
```
Comment (pernielsentikaer) [inline extensions/shopify-polaris-docs/README.md:1]:
```suggestion
# Shopify Polaris Docs
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/shopify-polaris-docs/README.md (hunk 2 above)
Comment (Rafi993) [reply extensions/shopify-polaris-docs/README.md:1]: Fixing casing

File: extensions/shopify-polaris-docs/package.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "shopify-polaris-docs",
+  "title": "Shopify-Polaris-Docs",
```
[the remark this answers is not in the document]
Comment (Rafi993) [reply extensions/shopify-polaris-docs/package.json:4]: fixing casing

File: extensions/shopify-polaris-docs/package.json [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  "author": "approxhuman",
+  "categories": [
+    "Documentation",
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search-polaris",
+      "title": "Search Polaris documentation",
```
[the remark this answers is not in the document]
Comment (Rafi993) [reply extensions/shopify-polaris-docs/package.json:16]: fixing description

File: extensions/shopify-polaris-docs/README.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+
+Search through Shopify Polaris documentation directlyfrom Raycast
+
+<img src="./demo1.png" width="500" height="300">
+
+<img src="./demo2.png" width="500" height="300">
```
[the remark this answers is not in the document]
Comment (Rafi993) [reply extensions/shopify-polaris-docs/README.md]: I have done change now

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**